main.go

```diff
@@ -0,0 +1,176 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+	"gopkg.in/yaml.v3"
+)
+
+func main() {
+	resolveMergeConflict()
+}
+
+func resolveMergeConflict() {
+	pendingMetadata := getPendingMetadata()
+	for testName, links := range pendingMetadata {
+		for _, link := range links {
+			toWPTMetadata(testName[1:], link)
+		}
+	}
+}
+
+// toWPTMetadata converts a pending metadata entry to a wpt-metadata META.yml
+// entry and writes it to disk.
+//
+// Directories and META.yml files are created as needed, or an existing one may
+// be updated.
+func toWPTMetadata(testName string, link shared.MetadataLink) {
+	dirpath := path.Dir(testName)
+	testpath := path.Base(testName)
+	createDirIfNeeded(dirpath)
+
+	// Open the META.yml file that we would write to, if it exists. In the case
+	// where it does exist we don't want to overwrite the file, so open in
+	// read-write-or-create mode.
+	filepath := dirpath + "/META.yml"
+	f, err := os.OpenFile(filepath, os.O_RDWR|os.O_CREATE, 0755)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+
+	fi, err := f.Stat()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if fi.Size() == 0 {
+		// This is an entirely new META.yml file, so create a new
+		// shared.Metadata representation for our expectation and write it out.
+		fmt.Println("CREATED-META-YML: for test", testName, "with url", link.URL)
+		metadata := shared.Metadata{
+			Links: []shared.MetadataLink{
+				{
+					Product: link.Product,
+					URL:     link.URL,
+					Results: []shared.MetadataTestResult{{
+						TestPath: testpath,
+					}},
+				},
+			},
+		}
+		writeToFile(metadata, f)
+		return
+	}
+
+	data, err := ioutil.ReadAll(f)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var metadata shared.Metadata
+	err = yaml.Unmarshal(data, &metadata)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// First check whether we have an existing crbug link in this META.yml; if
```

Comment:
```suggestion
	// First check whether we have an existing link in this META.yml; if
```

(Might not be a crbug.com link.)